--- changelog	2013-10-18 15:17:57.640081683 +0200
+++ changelog.wheezy	2013-10-18 15:19:58.442690776 +0200
@@ -1,3 +1,10 @@
+alternc (3.2) stable; urgency=low
+
+  * Version identical to 3.1 for Squeeze
+  * Includes a small dovecot patch / dependency for dovecot 2.0 for Wheezy
+
+ -- Benjamin Sonntag <benjamin@sonntag.fr>  Thu, 18 Oct 2013 15:19:00 +0200
+
 alternc (3.1) stable; urgency=low
 
   * feature: import of existing DNS zone
