--- control.squeeze	2017-10-06 12:01:52.272243664 +0200
+++ control	2017-10-06 12:03:02.016307914 +0200
@@ -38,16 +38,19 @@
        , sudo
        , adduser
        , dnsutils
-       , dovecot-common (>=1:1.2.15)
-       , dovecot-common(<< 1:2.0)
-       , dovecot-imapd (>= 1:1.2.15)
-       , dovecot-pop3d (>= 1:1.2.15)
+       , dovecot-common (>=1:2.1.7)
+       , dovecot-imapd
+       , dovecot-pop3d
+       , dovecot-mysql
        , vlogger
        , mailutils | mailx
        , zip
        , incron
        , cron
        , opendkim
+       , opendkim-tools
+       , dovecot-sieve
+       , dovecot-managesieved
        , mysql-client(>= 5.0) | mariadb-client
        , php5-curl
        , quota
@@ -123,15 +126,18 @@
          , gettext (>= 0.10.40-5)
          , adduser
          , sudo
-         , dovecot-common (>=1:1.2.15)
-         , dovecot-common(<< 1:2.0)
-         , dovecot-imapd (>= 1:1.2.15)
-         , dovecot-pop3d (>= 1:1.2.15)
+         , dovecot-common (>=1:2.1.7)
+         , dovecot-imapd
+         , dovecot-pop3d
+         , dovecot-mysql
          , vlogger
          , mailutils | mailx
          , incron
          , cron
          , opendkim
+         , opendkim-tools
+         , dovecot-sieve
+         , dovecot-managesieved
          , mysql-client(>= 5.0) | mariadb-client
          , php5-curl
          , ${misc:Depends}
