--- control.squeeze	2014-09-29 16:44:01.334940844 +0200
+++ control	2014-09-29 16:31:44.088155733 +0200
@@ -39,15 +39,18 @@
        , adduser
        , mysql-client
        , dnsutils
-       , dovecot-common (>=1:1.2.15)
-       , dovecot-common(<< 1:2.0)
-       , dovecot-imapd (>= 1:1.2.15)
-       , dovecot-pop3d (>= 1:1.2.15)
+       , dovecot-common (>=1:2.1.7)
+       , dovecot-imapd
+       , dovecot-pop3d
+       , dovecot-mysql
        , vlogger
        , mailutils | mailx
        , incron
        , cron
        , opendkim
+       , opendkim-tools
+       , dovecot-sieve
+       , dovecot-managesieved
        , mysql-client(>= 5.0)
        , php5-curl
 Recommends: mysql-server(>= 5.0)
@@ -123,16 +126,20 @@
          , adduser
          , mysql-client
          , sudo
-         , dovecot-common (>=1:1.2.15)
-         , dovecot-common(<< 1:2.0)
-         , dovecot-imapd (>= 1:1.2.15)
-         , dovecot-pop3d (>= 1:1.2.15)
+         , dovecot-common (>= 1:2.1.7)
+         , dovecot-imapd
+         , dovecot-pop3d
+         , dovecot-mysql
          , vlogger
          , mailutils | mailx
          , incron
          , cron
          , opendkim
-         , mysql-client(>= 5.0)
+         , opendkim-tools
+         , dovecot-managesieved
+         , dovecot-sieve
+         , dovecot-mysql
+         , mysql-client(>= 5.0)
          , php5-curl
 Recommends:
          quota
