--- control.squeeze	2016-05-27 14:30:23.182580393 +0200
+++ control	2016-05-27 14:32:08.785264702 +0200
@@ -39,16 +39,19 @@
        , adduser
        , mysql-client
        , dnsutils
-       , dovecot-common (>=1:1.2.15)
-       , dovecot-common(<< 1:2.0)
-       , dovecot-imapd (>= 1:1.2.15)
-       , dovecot-pop3d (>= 1:1.2.15)
+       , dovecot-common (>=1:2.1.7)
+       , dovecot-imapd
+       , dovecot-pop3d
+       , dovecot-mysql
        , vlogger
        , mailutils | mailx
        , zip
        , incron
        , cron
        , opendkim
+       , opendkim-tools
+       , dovecot-sieve
+       , dovecot-managesieved
        , mysql-client(>= 5.0)
        , php5-curl
        , quota
@@ -124,15 +127,18 @@
          , adduser
          , mysql-client
          , sudo
-         , dovecot-common (>=1:1.2.15)
-         , dovecot-common(<< 1:2.0)
-         , dovecot-imapd (>= 1:1.2.15)
-         , dovecot-pop3d (>= 1:1.2.15)
+         , dovecot-common (>=1:2.1.7)
+         , dovecot-imapd
+         , dovecot-pop3d
+         , dovecot-mysql
          , vlogger
          , mailutils | mailx
          , incron
          , cron
          , opendkim
+         , opendkim-tools
+         , dovecot-sieve
+         , dovecot-managesieved
          , mysql-client(>= 5.0)
          , php5-curl
          , ${misc:Depends}
